cloud_publisher:
  ros__parameters:
    cloud_topic: "/ply_cloud"
    input_path: "/tmp/clouds"
